config.go

```diff
@@ -1 +1,105 @@
 package main
+
+import (
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	flags "github.com/btcsuite/go-flags"
+	"path/filepath"
+	"io/ioutil"
+	"os"
+)
+
+const (
+	defaultConfigFilename = "lnwallet.conf"
+	defaultDataDirname    = "test_wal"
+	defaultRPCPort        = 10009
+	defaultSPVMode        = false
+	defaultPeerPort       = 10011
+	defaultBTCDHost       = "localhost:18334"
+	defaultBTCDUser       = "user"
+	defaultBTCDPass       = "passwd"
+	defaultBTCDCACertPath = ""
+	defaultUseRegtest     = false
+	defaultSPVHostAdr     = "localhost:18333"
+	defaultBTCDNoTLS      = false
+)
+
+var (
+	lnwalletHomeDir   = btcutil.AppDataDir("lnwallet", false)
+	defaultConfigFile = filepath.Join(lnwalletHomeDir, defaultConfigFilename)
+	defaultDataDir    = filepath.Join(lnwalletHomeDir, defaultDataDirname)
+)
+
+type config struct {
```

Comment:
lnwallet's configuration struct should be embedded within this lnd-level config struct. lnwallet's is rather bare atm, but in the near future it will house more wallet specific details such as the the default CSV timeout, how fees will be distributed, policy related to funding transactions etc. As a result, you'll be able to eliminate most of the fields here below as they're already listed within `lnwallet.Config`.